src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -189,7 +280,7 @@ const ContentHighlightStepper = ({ enterpriseId }) => {
                 <Stepper.ActionRow.Spacer />
                 <Button
                   variant="tertiary"
-                  onClick={() => setCurrentStep(STEPPER_STEP_LABELS.SELECT_CONTENT)}
+                  onClick={handleNavigateFromConfirmContent}
```

Comment:
nit: Since this button navigates back to the "Select content" step, could it re-use the `handleNavigateToSelectContent` function? I believe because you're tracking which step the user is currently on as part of the event, re-using the same function would work? 🤔